api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -179,7 +179,9 @@ public static class WgsCohortExtractionConfig {
     public Integer extractionMethodLogicalVersion;
     public String extractionDestinationDataset;
     // This should not exceed the value of GenomicExtractionService.MAX_EXTRACTION_SCATTER.
+    // TODO: remove?
```

Comment:
the new version chooses a scatter count with an optional override.  We may be able to stop computing this ourselves.

Reply:
shall we check with variant's team to see if this needed?

Reply:
yes